src/MiniExcel.Csv/GlobalUsings.cs

```diff
@@ -0,0 +1,16 @@
+// Global using directives
+
+global using System.Collections;

```

Comment:
The latest practice is to put these in csproj

https://github.com/dotnet/efcore/blob/05fe6269fe1df70267bfe7cb04062b6f9dbe02ab/src/EFCore/EFCore.csproj#L21-L42